### PR TITLE
style: run prettier on Glossary

### PR DESCRIPTION
**Pull request by nschonni**

### CODE REVIEW

File: files/en-us/glossary/semantics/index.md [hunk 1]
```diff
@@ -31,7 +31,9 @@ By default, most browser's [user agent stylesheet](/en-US/docs/Web/CSS/Cascade#u
 On the other hand, you could make any element _look_ like a top level heading. Consider the following:
 
 ```html
-<span style="font-size: 32px; margin: 21px 0;">Is this a top level heading?</span>
+<span style="font-size: 32px; margin: 21px 0;"
+  >Is this a top level heading?</span
+>
```
Comment (nschonni) [inline files/en-us/glossary/semantics/index.md:36]: This one was the only part I think ended up worse, and I couldn't see a way of getting it any better without disabling it

File: files/en-us/glossary/base64/index.md [hunk 2]
```diff
@@ -153,17 +152,25 @@ function base64EncArr(aBytes) {
   let nUint24 = 0;
   for (let nIdx = 0; nIdx < nLen; nIdx++) {
     nMod3 = nIdx % 3;
-    if (nIdx > 0 && (nIdx * 4 / 3) % 76 === 0) { 
+    if (nIdx > 0 && ((nIdx * 4) / 3) % 76 === 0) {
```
Comment (teoli2003) [inline files/en-us/glossary/base64/index.md:155]: This one is weird. Does Prettier really add parenthesis here? Any 10-year-old kid learns about `*` and `/` precedence!

File: files/en-us/glossary/semantics/index.md (hunk 1 above)
Comment (teoli2003) [inline files/en-us/glossary/semantics/index.md:36]: Alternative (The line is shorter with this text and doesn't wrap):
```suggestion
<span style="font-size: 32px; margin: 21px 0;">Not a top-level heading!</span>
```

See: https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAeAzgBwIZQATowCeANnALwA6IAZtDALToCWAXnEngMwBMmAHgG48AW2wAnAObMonHgEYBeAAyDqAPgByEGHmx4YETHjIA3OCTwALONgAmMyQEJUAeiy51IADQgjMZmh0ZFAJcQgAdwAFCQRglGwSCOwiYN8AI3FsMABrOBgAZWwROAAZGThkGkT0OAys3PyCnDBHZBhxAFc6kDgRdLg7O0HS3ElO7Ek4ADEIcTEYAKhJZBBsTsMfECsYERIAdStmeA8wOAK44+ZTY6JVsHQ0kBla8RgorMkxKpqegCt0PwCo4yABFTo6SpIaokWq+HDiV6rdLYAYkLaYcQyGD7Zh2GBWZAADmU8PCtX2WUwq0xcFe5i2AEcIfAPkZ4mt0AwoHBBoMtuI4MzmIKPpNvtDfr5aiJmO0uj0WMswSyoTC4SAYKjcfjCUgeL4OthmCRHABhCAiCW9dAAVi2nVqABVUfF1T1TN0AJJQYawApgLGYGAAQV9BWIZB+sLgAF9Y0A

File: files/en-us/glossary/falsy/index.md [hunk 3]
```diff
@@ -13,31 +13,39 @@ A **falsy** (sometimes written **falsey**) value is a value that is considered f
...
+if (0) {
+}
+if (-0) {
+}
+if (0n) {
+}
+if (NaN) {
+}
+if ("") {
+}
```
Comment (teoli2003) [inline files/en-us/glossary/falsy/index.md:48]: What about?

```suggestion
if (false) {
  // Not reachable
}

if (null) {
  // Not reachable
}

if (undefined) {
  // Not reachable
}

if (0) {
  // Not reachable
}

if (-0) {
  // Not reachable
}

if (0n) {
  // Not reachable
}

if (NaN) {
  // Not reachable
}

if ("") {
  // Not reachable
}
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
I proposed to fix the regression and found another place where it wasn't beautiful.

Happy to see more directories become Prettier-compliant.

File: files/en-us/glossary/base64/index.md (hunk 2 above)
Comment (nschonni) [reply files/en-us/glossary/base64/index.md:155]: yes, this was a prettier thing. People still regularly debat that stuff on Twitter though 😆

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**